src/main/resources/assets/farmersdelight/lang/zh_tw.json

```diff
@@ -29,13 +31,31 @@
   "block.farmersdelight.rice_bag": "米袋",
   "block.farmersdelight.straw_bale": "稻梗捆",
 
+  "block.farmersdelight.canvas_sign": "帆布牌",
+  "block.farmersdelight.white_canvas_sign": "白色帆布牌",
+  "block.farmersdelight.orange_canvas_sign": "橙色帆布牌",
+  "block.farmersdelight.magenta_canvas_sign": "洋紅色帆布牌",
+  "block.farmersdelight.light_blue_canvas_sign": "淺藍色帆布牌",
+  "block.farmersdelight.yellow_canvas_sign": "黃色帆布牌",
+  "block.farmersdelight.lime_canvas_sign": "淺綠色帆布牌",
+  "block.farmersdelight.pink_canvas_sign": "粉色帆布牌",
+  "block.farmersdelight.gray_canvas_sign": "灰色帆布牌",
+  "block.farmersdelight.light_gray_canvas_sign": "淺灰色帆布牌",
+  "block.farmersdelight.cyan_canvas_sign": "青色帆布牌",
+  "block.farmersdelight.purple_canvas_sign": "紫色帆布牌",
+  "block.farmersdelight.blue_canvas_sign": "藍色帆布牌",
+  "block.farmersdelight.brown_canvas_sign": "棕色帆布牌",
+  "block.farmersdelight.green_canvas_sign": "綠色帆布牌",
+  "block.farmersdelight.red_canvas_sign": "紅色帆布牌",
+  "block.farmersdelight.black_canvas_sign": "黑色帆布牌",
+
   "block.farmersdelight.tomatoes": "番茄",
   "block.farmersdelight.cabbages": "高麗菜",
   "block.farmersdelight.onions": "洋蔥",
-  "block.farmersdelight.rice_crop": "稻穗",
-  "block.farmersdelight.rice_upper_crop": "稻穗",
-  "block.farmersdelight.brown_mushroom_colony": "棕色蘑菇菌落",
-  "block.farmersdelight.red_mushroom_colony": "紅色蘑菇菌落",
+  "block.farmersdelight.rice": "稻穗",
```

Comment:
Following up:

- `rice` reverts to `rice_crop`;
- `rice_panicles` reverts to `rice_upper_crop`.